Makefile

```diff
@@ -29,11 +29,23 @@ DIST_DIRS         = find * -type d -exec
 GOOS ?= $(shell go env GOOS)
```

Comment:
something should be updtaed for line 26?
TARGETS ?= darwin/amd64 linux/amd64 linux/386 linux/arm linux/arm64 linux/ppc64le

Reply:
> and if you could provide some dev doc to help the understanding, it will be better :)
I will send another patch for doc later, once all these implementations got approved. 
> 
> I also got this error after try some commands:
> 
You should use make upload-image-manila-csi-plugin to include go build. 
Also you can override ALL_ARCH if you only wish to build and push specific set of image.
  
> make image-manila-csi-plugin
> find: ‘/root/go/src/github.com/cloud-provider-openstack/.go/src/k8s.io/cloud-provider-openstack’: No such file or directory
> cp manila-csi-plugin-amd64 cluster/images/manila-csi-plugin
> cp: cannot stat 'manila-csi-plugin-amd64': No such file or directory
> Makefile:182: recipe for target 'image-manila-csi-plugin' failed
It's complaining about can't found executable file from go build not found
> make: *** [image-manila-csi-plugin] Error 1

Reply:
Right, IIUC we should add linux/s390x here too

Reply:
done

> Right, IIUC we should add linux/s390x here too

Done